testing/functional/tests/cypress/integration/03-ministry/06-institution-note-restrictions.spec.ts

```diff
@@ -0,0 +1,611 @@
+    ministryUserViewsStudent.restrictionsSectionButton().click();
+    ministryUserViewsStudent.restrictionsSectionVerify().should("be.visible");
+    ministryUserViewsStudent.restrictionsSectionButton().click();
+    ministryUserViewsStudent.restrictionsSectionVerify().should("be.visible");
+    studentNoteRestrictionsObject
+      .addRestrictionsButton()
+      .should("be.visible")
+      .click();
+    studentNoteRestrictionsObject.addNewRestrictionsText().should("be.visible");
+    studentNoteRestrictionsObject.cancelButton().click();
+    studentNoteRestrictionsObject.allRestrictions().should("be.visible");
+  });
+
+  it("Check that all mandatory fields have been filled out in Institution Restriction section.", () => {
+    cy.fixture("ministryAddNewRestrictionsInInstitution").then((_testData) => {
+      cy.intercept("GET", "**/options-list").as("options-list");
+      cy.intercept("GET", "**/Designation").as("Designation");
+      ministryCustomCommand.loginMinistry();
+      dashboardMinistryObject.dashboardText().should("be.visible");
+      dashboardMinistryObject.searchInstitutionsText().click();
+      ministryUserViewsInstitution.operatingNameInputText().type("a");
+      ministryUserViewsStudent.searchButton().eq(1).click();
+      ministryUserViewsInstitution.viewButtonFirstRowInstitution().click();
+      ministryUserViewsStudent.restrictionsSectionButton().click();
+      ministryUserViewsStudent.restrictionsSectionVerify().should("be.visible");
+      ministryUserViewsStudent.restrictionsSectionButton().click();
+      ministryUserViewsStudent.restrictionsSectionVerify().should("be.visible");
+      studentNoteRestrictionsObject
+        .addRestrictionsButton()
+        .should("be.visible")
+        .click();
+      studentNoteRestrictionsObject
+        .addNewRestrictionsText()
+        .should("be.visible");
+      cy.wait("@options-list");
+      studentNoteRestrictionsObject.restrictionsDropdown().eq(0).click();
+      studentNoteRestrictionsObject
+        .restrictionsValue()
+        .eq(1)
+        .type(_testData.category)
+        .type("{enter}");
+      cy.wait("@Designation");
+      studentNoteRestrictionsObject.addRestrictionButtonDialogBox().click();
+      studentNoteRestrictionsObject.reasonErrorMessage().should("be.visible");
+      studentNoteRestrictionsObject.clearButton().click();
+      studentNoteRestrictionsObject.restrictionsDropdown().eq(2).click();
+      studentNoteRestrictionsObject
+        .restrictionsValue()
+        .eq(3)
+        .type(_testData.reason)
+        .type("{enter}");
+      studentNoteRestrictionsObject.addRestrictionButtonDialogBox().click();
+      studentNoteRestrictionsObject.categoryErrorMessage().should("be.visible");
+      studentNoteRestrictionsObject.clearButton().click();
+    });
+  });
+
+  it("Verify that that the Reason dropdown should be empty without filling in the Category in Institution Restriction section.", () => {
```

Comment:
Please remove that double _that_ from the test description.

Reply:
Text was removed.